src/views/domain-workflows/domain-workflows-table/domain-workflows-table.tsx

```diff
@@ -56,28 +49,19 @@ export default function DomainWorkflowsTable({ domain, cluster }: Props) {
     });
 
     if (errorPanelProps) {
-      return <ErrorPanel {...errorPanelProps} reset={refetch} />;
+      return (
+        <styled.ErrorPanelContainer>
+          <ErrorPanel {...errorPanelProps} reset={refetch} />
+        </styled.ErrorPanelContainer>
+      );
     }
   }
 
   return (
     <styled.TableContainer>
       <Table
         data={workflows}
-        columns={domainWorkflowsTableConfig}
```

Comment:
I could keep the columns field above and it would still keep all the columns unsortable if onSort isn't passed, but then it's less obvious why sortable columns are being rendered without the sort toggle.

Reply:
Changed this, let me know if this is okay or if we can tackle this differently.
Currently, I'm not a big fan of how we had to change the config typing to rectify this issue.